Multitenancy.md

```diff
@@ -375,3 +376,81 @@ curl -X POST "${ACAPY_ADMIN_URL}/multitenancy/wallet/{wallet_id}/remove" \
 ```jsonc
 {}
 ```
+
+### Per tenant settings
+
+To allow configurablity of ACA-Py startup parameters/environment variables at a tenant/subwallet level. [PR#2233](https://github.com/hyperledger/aries-cloudagent-python/pull/2233) will provide the ability to update the following subset of settings when creating or updating the subwallet:
+
+| Label  | Setting  |
+|---|---|
+| ACAPY_LOG_LEVEL  |  log.level |
```

Comment:
I believe that changing the log level will only take affect when writing to a physical log file is configured. It's not an issue, but probably wouldn't be clear to anyone. Maybe we just need a note about that limitation?

https://github.com/hyperledger/aries-cloudagent-python/blob/421c4ed6a3f750c7f30af7b85d1481a762a5acc4/aries_cloudagent/config/logging.py#L587